### DISCUSSION TITLE
Only 4/5 acheivement badges show on my profile

### DISCUSSION THREAD

**Discussion by Bazifrasool** [community/community#84967] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/45148731/1dbe0823-57c3-468b-8103-ffcf0c1f90f0)

**Comment by ghost** [comment, marked as answer]
Got to Profile settings in settings of your account. Then select the achievement visibility setting. There check whether every badge is selected to show on your profile.

All the best.
